src/chess/piece/Pawn.java

```diff
@@ -1,24 +1,81 @@
 package chess.piece;
 
-import java.util.HashMap;
-import java.util.Map;
+import chess.Position;
+
+import java.util.*;
```

Comment:
I'm not sure if here is aligned by IntelliJ automatically or not
(I mean IntelliJ put a blank line between own packages and standard packages?)

Plus, again if `*` is not placed by IntelliJ, I personally encourage you to list all dependencies you're using as studying perspective